pandas/tests/series/test_dtypes.py

```diff
@@ -48,8 +48,10 @@ def test_dtype(self, datetime_series):
 
         assert datetime_series.dtype == np.dtype('float64')
         assert datetime_series.dtypes == np.dtype('float64')
-        assert datetime_series.ftype == 'float64:dense'
-        assert datetime_series.ftypes == 'float64:dense'
+        with tm.assert_produces_warning(FutureWarning):
```

Comment:
can you add the comment with this issue number (and a blank line before)

Reply:
Sure.